.semaphore/semaphore.yml

```diff
@@ -52,4 +52,13 @@ blocks:
         commands:
           - docker exec -it semaphore-pyflask-docker_flasksemaphore_1 python -m unittest
 
-      
+# Automatically deploy to Heroku on successful builds on master branch:
+# For info on promotions, see:
+# https://docs.semaphoreci.com/article/50-pipeline-yaml#promotions
+promotions:
+  - name: Deploy to Heroku
+    pipeline_file: deploy-heroku.yml
+    auto_promote_on:

```

Comment:
I'd not do auto promotion here for the sake of people who'd be forking the repo and running it on Semaphore — they'd experience a failed workflow. Let's have manual here with a note on how to change to automatic.